tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/model/api/Categories.java

```diff
@@ -1,10 +1,7 @@
 package io.github.ygojson.tools.dataprovider.impl.yugipedia.model.api;
 
-import lombok.Data;
-
-@Data
 public class Categories {
 
-	private int ns;
-	private String title;
+	public int ns;
```

Comment:
I would say that it is better for all these models to make them private still and add the getters with the IDE